src/EdFi.AnalyticsMiddleTier.Tests/TestHarness.cs

```diff
@@ -324,6 +324,7 @@ void CreateSnapshot()
 FROM pathname
 ";
                     var filePath = connection.ExecuteScalar<string>(defaultFilePathForSqlDbFiles);
+                    filePath = @"C:\Projects\IndianaUEdFi\Ed-Fi-Analytics-Middle-Tier\snapshots";
```

Comment:
Please revert.

Reply:
@stephenfuqua Sorry about leaving extra code in here. I'll take it out. The language part, really was because I was not using the enterprise version of Visual Studio, and there was a part of the code that required 8.0. The project needed that to compile. I can try again without it so I can tell you what code was 8.0. I'll be making the changes shortly.